scripts/histmakers/mz_dilepton.py

```diff
@@ -0,0 +1,229 @@
+from utilities import boostHistHelpers as hh,common,output_tools
+
+parser,initargs = common.common_parser()
+
+import narf
+import wremnants
+from wremnants import theory_tools,syst_tools,theory_corrections, muon_validation, muon_calibration, muon_selections
+import hist
+import lz4.frame
+import logging
+import math
+import time
+import pdb
+
+parser.add_argument("-e", "--era", type=str, choices=["2016PreVFP","2016PostVFP"], help="Data set to process", default="2016PostVFP")
+parser.add_argument("--muonCorr", type=str, default="massfit", choices=["lbl", "trackfit_only_mctruth", "none", "massfit", "massfit_lbl", "trackfit_only"], 
+    help="Type of correction to apply to the muons")
+parser.add_argument("--muScaleMag", type=float, default=1e-4, help="Magnitude of dummy muon scale uncertainty")
+parser.add_argument("--muScaleBins", type=int, default=1, help="Number of bins for muon scale uncertainty")
+parser.add_argument("--muonCorrMag", default=1.e-4, type=float, help="Magnitude of dummy muon momentum calibration uncertainty")
+parser.add_argument("--muonCorrEtaBins", default=1, type=int, help="Number of eta bins for dummy muon momentum calibration uncertainty")
+parser.add_argument("--csvars_hist", action='store_true', help="Add CS variables to dilepton hist")
+parser.add_argument("--uncertainty-axes", type=str, nargs="*", default=["mll",], help="")
+parser.add_argument("--finePtBinning", action='store_true', help="Use fine binning for ptll")
+parser.add_argument("--bias-calibration", action='store_true', help="Adjust central value by calibration bias hist")
+parser.add_argument("--smearing", action='store_true', help="Smear pT such that resolution matches data")
+args = parser.parse_args()
+
+logging.basicConfig(level=logging.INFO)
+
+f = next((x for x in parser._actions if x.dest == "pt"), None)
+if f:
+    f.default = [34,26.,60.]
```

Comment:
Not very important, but args should be defined after changing the default for the pt range, or it will not be effective

Reply:
Good point, many thanks, this was actually also present in the mz_wlike histmaker script

Reply:
Right, I should be fixed since PR #123 for the mz_wlike, but I didn't check other similar scripts